base/stats.go

```diff
@@ -1322,167 +1418,167 @@ func (d *DbStats) initDatabaseStats() error {
 	labelKeys := []string{DatabaseLabelKey}
 	labelVals := []string{d.dbName}
 
-	resUtil.ReplicationBytesReceived, err = NewIntStat(SubsystemDatabaseKey, "replication_bytes_received", ReplicationBytesReceivedDesc, labelKeys, labelVals, prometheus.CounterValue, 0)
+	resUtil.ReplicationBytesReceived, err = NewIntStat(SubsystemDatabaseKey, "replication_bytes_received", StatUnitBytes, ReplicationBytesReceivedDesc, StatVersionAdded3dot1dot0, StatStabilityNoStability, labelKeys, labelVals, prometheus.CounterValue, 0)
```

Comment:
What makes this (and the other) stats have no stability, vs. committed?

Should no stability have a meaningful name, rather than just empty? (see prior `enum` comments)

Reply:
because not committed, this will be changed to one of the added enums